src/main.rs

```diff
@@ -75,6 +76,9 @@ enum Opt {
         )]
         sort: SortOrder,
 
+        #[arg(long, value_name = "FILTER")]
+        filter: Option<String>,
```

Comment:
Please add help text for this argument so that this sentence in the usage message remains accurate: https://github.com/dtolnay/cargo-llvm-lines/blob/0.4.19/src/main.rs#L33-L34. `--filter` is not a flag supported by `cargo build`.

Reply:
Oups forgot the doc, fixed